tests/unit/helpers/draftbills.test.js

```diff
@@ -1143,3 +994,41 @@ describe('getDraftBillsList', () => {
     );
   });
 });
+
+describe('getSurchargedPrice', () => {
+  const event = {
+    startDate: moment().hour(10),
+    endDate: moment().hour(16),
+  };
+
+  it('should return the price if the is no surcharge', () => {
```

Comment:
'should return the price if the**re** is no surcharge'

Reply:
oui